### ISSUE TITLE
How to build .net core 2.1

### ISSUE THREAD

**Issue by coffeymatt** [MicrosoftDocs/vsts-docs#800]
Hi,

I have a .net core build set up and working with .net core 2.0 apps. I'm wanting to move over to .net core 2.1.

At the moment, my updated branch doesn't build on visualstudio.com with the usual build steps:

https://docs.microsoft.com/en-us/vsts/build-release/apps/aspnet/build-aspnet-core?view=vsts&tabs=gitvsts%2Cweb%2Cdeploy-windows

I get:

C:\Program Files\dotnet\sdk\2.1.105\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.TargetFrameworkInference.targets(135,5): error : The current .NET SDK does not support targeting .NET Core 2.1.  Either target .NET Core 2.0 or lower, or use a version of the .NET SDK that supports .NET Core 2.1. 

Is there anything I can do to support more recent versions of .NET core on the build agent? If so, perhaps this could be included in the docs?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 483dcb1c-1a89-2b20-4cf2-abd3ae88d9ed
* Version Independent ID: dc4722ed-5a4b-26ec-e7d5-a149bf6e79a7
* Content: [CI build for an ASP.NET Core app](https://docs.microsoft.com/en-us/vsts/build-release/apps/aspnet/build-aspnet-core?view=vsts&tabs=gitvsts%2Cweb%2Cdeploy-windows)
* Content Source: [docs/build-release/apps/aspnet/build-aspnet-core.md](https://github.com/MicrosoftDocs/vsts-docs/blob/live/docs/build-release/apps/aspnet/build-aspnet-core.md)
* Service: **devops**
* GitHub Login: @andyjlewis
* Microsoft Alias: **alewis**

**Comment by vijayma** [comment]
We will support .net core 2.1 once it has RTMed. We do not install preview versions on hosted agents. You can use a private agent and work around this problem.

**Comment by coffeymatt** [comment]
Ok thank

**Comment by allenbrubaker** [comment]
It's released... when can we expect this support for v2.1 in the build agent?

**Comment by vijayma** [comment]
We have started the rollout of updated image in hosted agents. It should be everywhere before the end of this week. Thanks.

**Comment by daniloalsilva** [comment]
Still not working on agents, any update about rollout to .net core 2.1?

**Comment by Dushelov** [comment]
Same question. When .net core 2.1 will be supported by hosted agents?

**Comment by imranmomin** [comment]
Still not working for me. My account is located at **South Central US** region

**Comment by Liero** [comment]
I was able to build asp.net core 2.1 app using hosted build aged (West Europe). My first build task is:
.NET Core Tool Installer: 
![image](https://user-images.githubusercontent.com/3494407/41403082-492b9a00-6fc4-11e8-8641-90195c7b6797.png)

I got runtime version from [here](https://github.com/dotnet/core/blob/master/release-notes/releases.csv).

My question is:
Is .NET Core Tool Installer task required on hosted build agent?
